modules/computer.c

```diff
@@ -280,7 +280,7 @@ void scan_dev(gboolean reload)
        { N_("C (Clang)"), "clang -v", "\\d+\\.\\d+", FALSE },
        { N_("D (dmd)"), "dmd --help", "\\d+\\.\\d+", TRUE },
        { N_("Gambas3 (gbc3)"), "gbc3 --version", "\\d+\\.\\d+\\.\\d+", TRUE },
-       { N_("Java"), "javac -version", "\\d+\\.\\d+\\.\\d+", FALSE },
+       { N_("Java"), "javac -version", "(?<=javac )(\\d+.?)+", TRUE },
```

Comment:
Java is a mess. Recent versions (checked on 11) report version to stdout while older (checked 6 and 8) write it to stderr. But taking into account that 11 is only LTS available (for "casual" users) and java 8 have no public updates anymore, should be safe to use newer way from now.

Regex changed because I've seen `javac 11.0.9.1` on centos 7.

Reply:
almost 90% of 3trh party java progams does not runs with "only LTS available" seems you are talking only about openjdk right?

Reply:
almost 90% of 3trh party java progams does not runs with "only LTS available", almost java 8 are commonly used today, seems you are talking only about openjdk right?

Reply:
In my experience there is only single `java` binary installed inside /bin or /usr/bin in various distributions. 
Played with centos:7 container just to be sure. `/usr/bin/java` symlinked to `/etc/alternatives/java` which is symlinked to one of javas inside `/usr/lib/jvm`.